armada_jupyter/armada.py

```diff
@@ -0,0 +1,69 @@
+"""
+Example of getting jupyter notebook running on a k8s cluster.
+"""
+
+
+import grpc
+from armada_client.client import ArmadaClient
+from armada_client.armada import submit_pb2
+
+from armada_jupyter.constants import HOST, PORT, DISABLE_SSL
+
+
+def create_jupyter_service():
+    """
+    Create a jupyter service.
+    """
+
+    return submit_pb2.ServiceConfig(
```

Comment:
We have an issue to make these more configurable also?

Reply:
https://github.com/G-Research/armada-jupyter/issues/40

This was in the future - considerations in the design doc, so definitely not happening in the short term, but yes its definitely a needed feature.